src/routes/online-lecture.js

```diff
@@ -50,35 +44,58 @@ export default () => {
       <div>- 2분 쿨타임 제거: 2분 쿨타임을 제거할 수 있습니다. 단, 동시에 여러 콘텐츠 학습을 하지 않도록 주의해 주세요.</div>
       <div>- 강의 숨기기: 진도율 100%인 강의를 숨길 수 있습니다.</div>
       <div style="margin-top: 20px">
-        <button type="button" id="btn-cooltime" class="btn2 btn-learn">2분 쿨타임 제거</button>
+        <button type="button" id="${randomStr}" class="btn2 btn-learn">2분 쿨타임 제거</button>
         <button type="button" id="btn-hide-lecture" class="btn2 btn-gray">강의 숨기기 On / Off</button>
       </div>
     </div>
 `);
 
   // 2분 쿨타임 제거 버튼에 이벤트 설정
-  $('#btn-cooltime').click(() => {
-    appModule.getLrnStdSttus1 = function () {
-      axios.post('/std/lis/evltn/SelectLrnSttusStd.do', this.$data).then(function (response) {
-        this.lrnSttus = response.data;
+  $('#' + randomStr).click(() => {
+    let funcName;
```

Comment:
```suggestion
    let funcName = 'getLrnStdSttus1';
```

혹시 아래 로직에서 함수 이름을 찾지 못했을 경우를 대비해서 디폴트로 함수 이름을 넣어주는 것이 좀 더 안전할 것 같습니다.